Smooch.postman_collection.json

```diff
@@ -5639,7 +5668,7 @@
 								],
 								"body": {
 									"mode": "raw",
-									"raw": "{\n\t\"type\": \"messagebird\",\n    \"accessKey\": \"<messagebirdAccessKey>\",\n    \"originator\": \"<messagebirdOriginator>\"\n}"
+									"raw": "{\n\t\"type\": \"messagebird\",\n    \"accessKey\": \"<messagebirdAccessKey>\",\n    \"originator\": \"<messagebirdOriginator>\",\n    \"signingKey\": \"<messagebirdSigningKey>\"\n}"
```

Comment:
Should that also be added here as well?

https://github.com/smooch/smooch-postman-collection-public/pull/15/files#diff-4b081629b5ae173add1b20828fb6033bR5704